spec/index.bs

```diff
@@ -3811,7 +3816,17 @@ enum GPUPipelineStatisticName {
 };
 </script>
 
-When resolving pipeline statistics query, each result is written into uint64, and the number and order of the results written to GPU buffer matches the number and order of {{GPUPipelineStatisticName}} specified in {{GPUQuerySetDescriptor/pipelineStatistics}}.
+When resolving pipeline statistics query, each result is written into {{GPUSize64}}, and the number and order of the results written to GPU buffer matches the number and order of {{GPUPipelineStatisticName}} specified in {{GPUQuerySetDescriptor/pipelineStatistics}}.
+
+Pipeline statistics query requires {{GPUExtensionName/pipeline-statistics-query}} is available on the device.
+
+
+## Timestamp Query ## {#timestamp}
+Timestamp query allows application to write timestamp values to a {{GPUQuerySet}} by calling {{GPUProgrammablePassEncoder/writeTimestamp()}} on render pass or compute pass, and then resolve timestamp values in **nanoseconds** (type of {{GPUSize64}}) to a {{GPUBuffer}} (using {{GPUCommandEncoder/resolveQuerySet()}}).
```

Comment:
Please add an `Issue` that points to this PR and explains that because the feature provides high-resolution GPU timestamps we need to decide what constraints, if any, are on its availability.

Reply:
Done. 
Do we need to open an issue for further discussion?